ssh/tailssh/incubator.go

```diff
@@ -443,6 +465,7 @@ func (ss *sshSession) startWithPTY() (ptyFile *os.File, err error) {
 	cmd.Stdout = tty
 	cmd.Stderr = tty
 
+	ss.logf("starting pty command: %+v", cmd.Args)
```

Comment:
Excessively spammy? Or do we want this for an audit log?

Reply:
this is moving and not a new log line, I agree it's spammy but I don't want to drop it yet. Also it's not terrible because it's not like you get multiple ssh reqs/s